src/collection-browser.ts

```diff
@@ -652,18 +655,33 @@ export class CollectionBrowser
    */
   private get rightColumnTemplate(): TemplateResult {
     return html`
-      <div id="right-column" class="column">
-        <div id="cb-top-view">
-          <slot name="cb-top-slot"></slot>
-        </div>
-        ${this.isManageView
-          ? this.manageBarTemplate
-          : this.sortFilterBarTemplate}
-        <slot name="cb-results"></slot>
-        ${this.displayMode === `list-compact` && this.totalResults
-          ? this.listHeaderTemplate
+      <div
+        id="right-column"
+        class="column ${this.showSmartResults ? 'smart-results-spacing' : ''}"
+      >
+        ${this.showSmartResults
+          ? html`<div id="smart-results">
+              <slot name="smart-results"></slot>
+            </div>`
```

Comment:
Q: Is the wrapper div necessary here? Could we instead just apply the extra margin to the results section below?